.buildkite/pipeline.yml

```diff
@@ -44,15 +44,19 @@ steps:
 
         echo "--- :android: Build Android bundle"
         npm run bundle:android
+        node gutenberg/node_modules/react-native/scripts/compose-source-maps.js bundle/android/App.text.js.map bundle/android/App.js.map -o bundle/android/App.composed.js.map
```

Comment:
@SiobhyB I realized that source maps will only be needed for releases, so I'm going to limit the upload when building from a Git tag. Sorry for the last-minute change.

Reply:
Good catch! Makes total sense to limit the uploads to tags.